src/explore-education-statistics-admin/src/services/releaseDataFileService.ts

```diff
@@ -91,8 +85,8 @@ const releaseDataFileService = {
       `/release/${releaseId}/data/${dataFile.filename}/delete-plan?name=${dataFile.title}`,
     );
   },
-  deleteDataFiles(releaseId: string, dataFile: DataFile): Promise<null> {
-    return client.delete<null>(
+  deleteDataFiles(releaseId: string, dataFile: DataFile): Promise<boolean> {
```

Comment:
Don't think this API actually returns a boolean so this type is incorrect. We wouldn't want consumers to expect a boolean that will never materialise.

For DELETEs a void response is totally fine (a boolean by itself doesn't provide useful information). If the delete failed for whatever reason, the API would just return an error response (with appropriate status code) and that will subsequently throw an error in the frontend code.

Would revert this back to `Promise<void>`.

Reply:
Yep done - Thanks